### ISSUE TITLE
Timezone in string doesn't work/isn't blocked

### ISSUE THREAD

**Issue by ghost** [nickmaccarthy/python-datemath#21]
Hi,
I tried some examples with this library and it accepts timezones in the string but seems to ignore them:
```python
datemath("2016-01-01T16:20:00.5+12:00")
datetime.datetime(2016, 1, 1, 16, 20, 0, 500000, tzinfo=tzutc())
```
If you then try to do math on it it fails:
```python
datemath("2016-01-01T16:20:00.5+12:00||+1d")
datemath.helpers.DateMathException: Unable to calculate date: now: 2016-01-01T16:20:00.500000+00:00, offsetvalue: 1.0, unit: days - reason: 'Arrow' object has no attribute 'shift'
```
Can you please check if timezones in the string are supported and if not, block them?

datemath version: 1.5.1
python version: 3.7.1

PS: I'm aware of the tz parameter but was curious if it's supported inside the string

**Comment by nickmaccarthy** [comment]
hi @kaijm .  I think its possible to have both cases here.  I just submitted a new PR.

In the PR i am allowing the returned timeobj to be the TZ of the datetime string originall if there was a TZ offset in it.  If you had set both the TZ offset in your datetime string, the dateobj will be returned with that timezone.  If you had both a tz offset and were using the tz parameter, the tz offset in the string takes precedence. 

Would you mind testing out branch "issue_21" to see if that fixes your issues?

**Comment by ghost** [comment]
@nickmaccarthy it works great. Thanks a lot!

**Comment by nickmaccarthy** [comment]
Thanks for testing this out and bringing it my attention @kaijm .  This is now merged and updated to v1.5.2 in pypi now.

**Comment by nickmaccarthy** [comment]
fixed
